src/TimeService/timeservice-utils.h

```diff
@@ -0,0 +1,51 @@
+#ifndef TIMESERVICE_UTILS
+#define TIMESERVICE_UTILS
+
+#include <time.h>
+#include <stdint.h>
+
+#define NS_PER_SECOND (1000000000)
+
+#define CUC_TIME_ARRAY_SIZE 7
+
+typedef struct CUCTimestamp
+{
+   unsigned char pField;
+   unsigned char tField[CUC_TIME_ARRAY_SIZE];
+} CUCTimestamp;
+
+typedef struct CfsTimestamp
+{
+   uint64_t seconds;
+   uint64_t subseconds;
+} CfsTimestamp;
+
+typedef enum {
+    UNAVAILABLE = 0,
+    STOPPED = 1,
+    STARTED = 2
+} ClockStatusEnum;
+
+static int cmp_memory(const unsigned char *str1, const unsigned char *str2, int len)
```

Comment:
I would try to use full names of an arguments where it is possible (for example "length" not "len")

Reply:
OK